qb/config.params.sh

```diff
@@ -193,3 +193,4 @@ HAVE_NETWORK_VIDEO=no
 HAVE_STEAM=no              # Enable Steam build
 HAVE_ODROIDGO2=no          # ODROID-GO Advance rotation support (requires librga)
 HAVE_LIBSHAKE=no           # libShake haptic feedback support
+HAVE_CHECK=no              # check support for unit tests
```

Comment:
Perhaps we can just ignore the regular build system for this? I mean, these tests are more oriented to be run on a computer anyway, so we can just have a separate Makefile for tests (like Makefile.test) but do not include in anywhere else?
I'm also unsure on what's the best path here, but we'll figure it out :)

Reply:
How about only add the `test` target as a depency of `all` for debug builds?
Later if more is added to the unit tests, it would be good to have developers running the `test` target locally.

Reply:
How about only add the `test` target as a depency of `all` for debug builds?
Later if more is added to the unit tests, it would be good to have developers running the `test` target locally.